src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/DefendantResponseApplicantNotificationHandler.java

```diff
@@ -75,8 +89,27 @@ private CallbackResponse notifyApplicantSolicitorForDefendantResponse(CallbackPa
         );
     }
 
+    public Map<String, String> addPropertiesSpec(CaseData caseData, CallbackParams callbackParams) {
+        return Map.of(
+            CLAIM_LEGAL_ORG_NAME_SPEC, getLegalOrganisationName(caseData, callbackParams),
+            CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
+            RESPONDENT_NAME, getPartyNameBasedOnType(caseData.getRespondent1())
+        );
+    }
+
     private boolean isCcNotification(CallbackParams callbackParams) {
         return callbackParams.getRequest().getEventId()
             .equals(NOTIFY_APPLICANT_SOLICITOR1_FOR_DEFENDANT_RESPONSE_CC.name());
     }
+
+    //finding legal org name
+    private String getLegalOrganisationName(CaseData caseData, CallbackParams callbackParams) {
```

Comment:
If it is Applicant's org name. It worth renaming it as getApplicantsLegalOrgName